charts/jenkins/VALUES_SUMMARY.md

```diff
@@ -140,6 +140,14 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `controller.customInitContainers`     | Custom init-container specification in raw-yaml format | Not set                 |
 | `controller.sidecars.other`           | Configures additional sidecar container(s) for Jenkins controller | `[]`             |
 
+#### Kubernetes Pod Disruption Budget
+
+| Parameter                         | Description                          | Default                                   |
+| --------------------------------- | ------------------------------------ | ----------------------------------------- |
+| `controller.podDisruptionBudget.enabled` | Enable [Kubernetes Pod Disruption Budget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) configuration from `controller.podDisruptionBudget` (see below) | `false` |
+| `controller.podDisruptionBudget.maxUnavailable` | Number of pods that can be unavailable. Either an absolute number or a percentage. | `1` |
```

Comment:
you set it to 0 didn't you?

Reply:
I did... Though pending the conversation above there might not be a default here... 😉

Reply:
Default value removed in [`fa94718` (#461)](https://github.com/jenkinsci/helm-charts/pull/461/commits/fa94718414a334addc6e6fd39d80a9d2170317c4)